bundles/core/org.eclipse.smarthome.core.persistence/src/main/java/org/eclipse/smarthome/core/persistence/rest/ServiceBean.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2010-2013, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.persistence.rest;
+
+/**
+ * This is a java bean that is used with JAXB to serialize items to JSON.
+ *
+ * @author Chris Jackson - Initial Contribution
+ *
+ */
+public class ServiceBean {
+    public String name;
```

Comment:
We should probably work on the persistence services in future to also have some more meta-data like a localizable label and description for the UI...